packages/terra-button-group/src/ButtonGroupButton.scss

```diff
@@ -1,14 +1,55 @@
 :local {
   .button-group-button {
+    background-color: var(--terra-button-group-button-background-color, #fff);
+    border-color: var(--terra-button-group-button-border-color, #c8cacb);
     border-radius: 0;
     border-style: solid;
     border-width: 1px;
+    box-shadow: var(--terra-button-group-button-box-shadow, none);
+    color: var(--terra-button-group-button-color, #434a4d);
+    font-weight: var(--terra-button-group-button-font-weight, 400);
     margin: 0;
     position: relative;
 
+    &.is-disabled,
+    &:disabled {
+      opacity: var(--terra-button-group-button-disabled-opacity, 0.3);
+    }
+
+    &.is-focused,
     &:focus {
+      background-color: var(--terra-button-group-button-focus-background-color, #fff);
+      border-color: var(--terra-button-group-button-focus-border-color, #c8cacb);
+      border-width: var(--terra-button-group-button-focus-border-width, 1px);
```

Comment:
I'm seeing some weird resizing/shifting happening on your mock theme. I'm pretty sure it's because of changing the border width on focus. I'm wondering if we should try to achieve this style with an inset box-shadow instead.

Reply:
Good catch. I was trying to mimic the consumer theme by using a heavier border width for the focus style but this has the side effect of a jumpy screen. Using an inset box-shadow does work better. https://github.com/cerner/terra-core/pull/1155/commits/553693e81282805d5c9d4db1168757c4fcbca411